tools/gometalinter.json

```diff
@@ -2,7 +2,7 @@
     "Linters": {
         "vet": "go tool vet -composites=false :PATH:LINE:MESSAGE"
    },
-    "Enable": ["golint", "vet", "ineffassign", "unconvert", "misspell"],
+    "Enable": ["golint", "vet", "ineffassign", "misspell"],
```

Comment:
I do not understand why switching to `go mod` triggers the `unconvert` linter. Will check tomorrow.